### ISSUE TITLE
issue when running bundle install --local: NoMethodError: undefined method 'contains_requirable_file?‘ fornil:NilClass

### ISSUE THREAD

**Issue by guosw1990** [bundler/bundler#5147]
I ran the command 'bundle install --local' and expected to bundle my rails application. Instead, what actually happened was error report details below:

 ```
NoMethodError: undefined method `contains_requirable_file?' for nil:NilClass
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:927:in `block in find_by_path'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:895:in `block in each'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:894:in `each'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:894:in `each'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:926:in `find'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/specification.rb:926:in `find_by_path'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems.rb:190:in `try_activate'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:126:in `rescue in require'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendor/net/http/persistent.rb:12:in `<top (required)>'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendored_persistent.rb:12:in `<top (required)>'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
      C:/Ruby22-x64/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/fetcher.rb:2:in `<top (required)>'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/cli/install.rb:93:in `run'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/cli.rb:175:in `install'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/cli.rb:11:in `start'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/exe/bundle:27:in `block in <top (required)>'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/lib/bundler/friendly_errors.rb:98:in `with_friendly_errors'
      C:/Ruby22-x64/lib/ruby/gems/2.2.0/gems/bundler-1.12.5/exe/bundle:19:in `<top (required)>'
      C:/Ruby22-x64/bin/bundle:23:in `load'
      C:/Ruby22-x64/bin/bundle:23:in `<main>'

Environment

  Bundler   1.12.5
  Rubygems  2.4.5.1
  Ruby      2.2.5p319 (2016-04-26 revision 54774) [x64-mingw32]
  Git       2.9.0.windows.1
--- TEMPLATE END ----------------------------------------------------------------

Unfortunately, an unexpected error occurred, and Bundler cannot continue.

First, try this link to see if there are any existing issue reports for this error:
https://github.com/bundler/bundler/search?q=undefined+method+%60contains_requirable_file%3F%27+for+nil%3ANilClass&type=Issues

If there aren't any reports for this error yet, please create copy and paste the report template above into a new issue. Don't forget to anonymize any private data! The new issue form is located at:
https://github.com/bundler/bundler/issues/new
```

**Comment by segiddins** [comment]
This potentially looks like a bug in RubyGems to me, but that RubyGems version is well over a year old. Can you check if it's fixed with 2.6.8? Thanks!

**Comment by lynnco** [comment]
@guosw1990

> This potentially looks like a bug in RubyGems to me, but that RubyGems version is well over a year old. Can you check if it's fixed with 2.6.8? Thanks!

---

Closing due to waiting on user response. Feel free to ping a team member to re-open this issue if needed!
